src/ert/storage/local_ensemble.py

```diff
@@ -609,14 +616,27 @@ def load_responses(self, key: str, realizations: Tuple[int]) -> xr.Dataset:
             Loaded xarray Dataset with responses.
         """
 
-        if key not in self.experiment.response_configuration:
+        select_key = False
+        if key in self.experiment.response_configuration:
+            response_type = key
+        elif key not in self.experiment.response_key_to_response_type:
             raise ValueError(f"{key} is not a response")
+        else:
+            response_type = self.experiment.response_key_to_response_type[key]
+            select_key = True
+
         loaded = []
         for realization in realizations:
-            input_path = self._realization_dir(realization) / f"{key}.nc"
+            input_path = self._realization_dir(realization) / f"{response_type}.nc"
```

Comment:
Regarding our offline discussion about new response types (CSV_RESPONSE), would we require something else here?

Reply:
I think so yes, not 100% on exactly what. But we can set the response_type (or some other thing to identify it) dynamically when loading data from runpath, as then we will know the primary key of the data, which is part of the dynamic response type. Should be possible to make it fit later on